src/transforms/textDecorationLine.js

```diff
@@ -0,0 +1,20 @@
+const { tokens } = require('../tokenTypes')
+
+const { SPACE, LINE } = tokens
+
+module.exports = tokenStream => {
+  const lines = []
+
+  let didParseFirst = false
+  while (tokenStream.hasTokens()) {
+    if (didParseFirst) tokenStream.expect(SPACE)
+
+    lines.push(tokenStream.expect(LINE))
+
+    didParseFirst = true
+  }
+
+  lines.sort().reverse()
```

Comment:
oh this is clever trick, though it would not work if there were other values supported :)

Reply:
Oh that's true. How would we solve that though. What if RN added a new value that was meant to be put after the current last value?

Reply:
You could use the compare function from here to only reorder those two values but not touch the other ones:
https://github.com/styled-components/css-to-react-native/issues/64#issuecomment-363865691

Reply:
at least that way it's somewhat future proof

Reply:
ping @jacobp100 do you think that this should be changed or left like it is now?

Reply:
I'll probably leave as-is. Tbh I think they'd switch to an array of values if they added new values (like `fontVariant`)